src/mame/drivers/astrcorp.cpp

```diff
@@ -592,6 +590,10 @@ void astrocorp_state::skilldrp(machine_config &config)
 	TICKET_DISPENSER(config, m_hopper, attotime::from_msec(200), TICKET_MOTOR_ACTIVE_HIGH, TICKET_STATUS_ACTIVE_LOW );
 
 	/* video hardware */
+	/*
```

Comment:
You're missing the inlined TODO now. 
Suggest refactor of this C-style comment among the lines of:
```
		// TODO: understand if later hardware uses different screen params
	    // XTAL is almost surely NOT 20 MHz.
	    // Also weirdly enough, there's an unused 6x PAL XTAL according to notes, but VSync is 58,85 Hz?
```